flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql2rel/RelDecorrelator.java

```diff
@@ -268,20 +270,20 @@ protected RelNode decorrelate(RelNode root) {
                                                         .FilterIntoJoinRuleConfig.class)
                                         .toRule())
                         .addRuleInstance(
-                                CoreRules.FILTER_PROJECT_TRANSPOSE
-                                        .config
-                                        .withRelBuilderFactory(f)
-                                        .as(FilterProjectTransposeRule.Config.class)
-                                        .withOperandFor(
-                                                Filter.class,
-                                                filter ->
-                                                        !RexUtil.containsCorrelation(
-                                                                filter.getCondition()),
-                                                Project.class,
-                                                project -> true)
-                                        .withCopyFilter(true)
-                                        .withCopyProject(true)
-                                        .toRule())
+                                FlinkFilterProjectTransposeRule.build(
```

Comment:
Since we have added new changes, the section in the class comment marking the scope of the line changes should also be updated.